### PR TITLE
fix(links): update github links of user authentication overview

### PR DESCRIPTION
**Pull request by redboul**
Closes [BS-15570](https://bonitasoft.atlassian.net/browse/BS-15570)

### CODE REVIEW

File: md/user-authentication-overview.md [hunk 1]
```diff
@@ -45,41 +44,41 @@ username, password, tenant id (if provided in the URL, only applied to
...
-10. `authenticate` method is called on the [AuthenticationManager](https://github.com/bonitasoft/bonita-web/blob/7.0.0/common/src/main/java/org/bonitasoft/console/common/server/auth/AuthenticationManager.java).
-`authenticate` method of [StandardAuthenticationManagerImpl](https://github.com/bonitasoft/bonita-web/blob/7.0.0/common/src/main/java/org/bonitasoft/console/common/server/auth/impl/standard/StandardAuthenticationManagerImpl.java)
+7. URL `/loginservice` is handled by [LoginServlet](https://github.com/bonitasoft/bonita-web/blob/${varVersion}.0/server/src/main/java/org/bonitasoft/console/common/server/login/servlet/LoginServlet.java).
+8. [LoginServlet](https://github.com/bonitasoft/bonita-web/blob/${varVersion}.0/server/src/main/java/org/bonitasoft/console/common/server/login/servlet/LoginServlet.java)
+calls `login` method of [LoginManager](https://github.com/bonitasoft/bonita-web/blob/${varVersion}.0/server/src/main/java/org/bonitasoft/console/common/server/login/LoginManager.java)
+9. [LoginManager](https://github.com/bonitasoft/bonita-web/blob/${varVersion}.0/common/src/main/java/org/bonitasoft/console/common/server/login/LoginManager.java)
```
Comment (cpuy) [inline md/user-authentication-overview.md]: Wrong Url

File: md/user-authentication-overview.md [hunk 2]
```diff
@@ -45,41 +44,41 @@ username, password, tenant id (if provided in the URL, only applied to
...
+implementation based on configuration file for current tenant ([`authenticationManager-config.properties`](BonitaBPM_platform_setup.md)). The default [AuthenticationManager](https://github.com/bonitasoft/bonita-web/blob/${varVersion}.0/common/src/main/java/org/bonitasoft/console/common/server/auth/AuthenticationManager.java)
+is [StandardAuthenticationManagerImpl](https://github.com/bonitasoft/bonita-web/blob/${varVersion}.0/server/src/main/java/org/bonitasoft/console/common/server/auth/impl/standard/StandardAuthenticationManagerImpl.java).
+10. `authenticate` method is called on the [AuthenticationManager](https://github.com/bonitasoft/bonita-web/blob/${varVersion}.0/common/src/main/java/org/bonitasoft/console/common/server/auth/AuthenticationManager.java).
+`authenticate` method of [StandardAuthenticationManagerImpl](https://github.com/bonitasoft/bonita-web/blob/${varVersion}.0/common/src/main/java/org/bonitasoft/console/common/server/auth/impl/standard/StandardAuthenticationManagerImpl.java)
```
Comment (cpuy) [inline md/user-authentication-overview.md]: Wrong URL

**Comment by cpuy** [comment]
Two links broken dude.

What a mess !!!
IMHO we should definitively extract LoginManager and so on in a dedicated artifact that user can find in maven central and also all these informations should be in Java doc

**Comment by redboul** [comment]
Do you suggest to drop this fix?

**Comment by cpuy** [comment]
Nop just fix the links and I'll merge.
Just a though

**Pull request merged**